pages/tokens/[id]/bid.tsx

```diff
@@ -120,6 +140,8 @@ const BidPage: NextPage<Props> = ({ now, assetId, meta, currentAccount }) => {
   })
   useExecuteOnAccountChange(refetch, ready)
 
+  const currencyRes = useChainCurrencies(data?.asset?.chainId, true)
```

Comment:
One suggestion to make it maybe more readable, not mandatory though. 
```suggestion
  const currencyRes = useChainCurrencies({ chainId: data?.asset?.chainId, onlyERC20: true })
```

Reply:
updated with `useChainCurrencies(chainId, { onlyERC20: true })` as `chainId` is required to be passed while `onlyERC20` is not